### PR TITLE
docs: cleanup docs and move into proper places

### PR DESCRIPTION
**Pull request by alanshaw**
* Moves `api` and `website` getting started documentation into the proper directories
* Adds overview documentation to the README at the root and instructions for running locally

### CODE REVIEW

File: packages/api/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+### Local env vars
+
+Inside the `/packages/api` folder create a file called `.env.local` with the following content.
+
+```ini
+SENTRY_TOKEN=<sentry user auth token>
+SENTRY_UPLOAD=false # toggle for sentry source/sourcemaps upload (capture will still work)
+```
+
+Production vars should set in Github Actions secrets.
```
Comment (vasco-santos) [inline packages/api/README.md:16]:
```suggestion
Production vars should be set in Github Actions secrets.
```

**Review by vasco-santos — COMMENTED** [review]
(just an initial pass, the set up ended up being long, so I will finish it tomorrow and report back)

File: packages/api/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+The nft.storage public API.
+
+## Getting Started
+
+### Local env vars
+
+Inside the `/packages/api` folder create a file called `.env.local` with the following content.
+
+```ini
```
[the remark this answers is not in the document]
Comment (vasco-santos) [reply packages/api/README.md:11]: What are the needed tokens permissions?

File: packages/api/README.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+Follow the quickstart guide to get an IPFS Cluster up and running: https://cluster.ipfs.io/documentation/quickstart/
+
+Expose the IPFS Proxy by opening the `docker-compose.yml` file and add:
+
+```yaml
+CLUSTER_IPFSPROXY_NODEMULTIADDRESS: /dns4/ipfs0/tcp/5001
+CLUSTER_IPFSPROXY_LISTENMULTIADDRESS: /ip4/0.0.0.0/tcp/9095
+```
+
+...to the `custer0` _environment_ and add:
```
Comment (vasco-santos) [inline packages/api/README.md:43]:
```suggestion
...to the `cluster0` _environment_ and add:
```

File: packages/api/README.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+# same as above
+wrangler kv:namespace create PINS --preview --env USER
+# same as above
+wrangler kv:namespace create FOLLOWUPS --preview --env USER
+# same as above
+wrangler kv:namespace create PINATA_QUEUE --preview --env USER
+# same as above
+```
+
+Go to `/packages/api/src/constants.js` _uncomment_ the first line and run `wrangler publish --env USER`.
```
Comment (vasco-santos) [inline packages/api/README.md:112]: In this step, I am getting:

```
Error: Something went wrong! Status: 400 Bad Request, Details {
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 10021,
      "message": "Uncaught ReferenceError: DEALS is not defined\n  at line 50606 in ./src/constants.js\n  at line 20 in __webpack_require__\n  at line 53167 in ./src/utils/debug.js\n  at line 20 in __webpack_require__\n  at line 54179\n  at line 54356 in ./src/utils/router.js\n  at line 20 in __webpack_require__\n  at line 50799 in ./src/index.js\n  at line 20 in __webpack_require__\n  at line 84\n"
    }
  ],
  "messages": []
}
```

I did the steps above to add the Deals (with success), but it does not feel it got applied. If I comment line 3 from constants, I get the same error for "USERS". How is this loading the references?

File: packages/api/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hugomrdias) [reply packages/api/README.md:11]: Tokens can be created here https://sentry.io/settings/account/api/auth-tokens/ and need the following scopes `event:admin, event:read, member:read, org:read, project:read, project:releases, team:read`

File: packages/api/README.md (hunk 4 above)
Comment (hugomrdias) [reply packages/api/README.md:112]: do you have the deals kv created in your CF account ?

**Review by hugomrdias — APPROVED** [review]

File: packages/api/README.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+```toml
+[env.USER]
+type = "webpack"
+name = "nft-storage-USER"
+account_id = "CF_ACCOUNT"
+workers_dev = true
+route = ""
+zone_id = ""
+vars = { ENV = "dev", DEBUG = "*", CLUSTER_API_URL = "", CLUSTER_IPFS_PROXY_API_URL = "" }
```
Comment (vasco-santos) [inline packages/api/README.md:83]:
```suggestion
vars = { ENV = "dev", DEBUG = "*", CLUSTER_API_URL = "", CLUSTER_IPFS_PROXY_API_URL = "", CLUSTER_ADDRS = "" }
```

File: packages/api/README.md (hunk 4 above)
Comment (vasco-santos) [reply packages/api/README.md:112]: After syncing with @alanshaw I could move forward on this one. Two things here:

1. create previous step where we add to the `wrangler.toml` file the kv_namespaces, or make it more clear on the comments above
2. constants.js file uncommented line also needs `SENTRY_DSN`

Comment (vasco-santos) [reply packages/api/README.md:112]: > do you have the deals kv created in your CF account ?

I was missing the `kv_namespaces`

Comment (hugomrdias) [reply packages/api/README.md:112]: the cli doesnt output instructions to add kv namespaces to the toml file ?

File: packages/api/README.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+kv_namespaces = []
+```
+
+Additionally, fill in the `CLUSTER_API_URL` and `CLUSTER_IPFS_PROXY_API_URL` with the localtunnel URLs you obtained when setting up the IPFS Cluster.
```
Comment (vasco-santos) [inline packages/api/README.md:87]: Please add an example below, given that we need to append `/api/v0/` to the `CLUSTER_IPFS_PROXY_API_URL`

File: packages/api/README.md (hunk 4 above)
Comment (vasco-santos) [reply packages/api/README.md:112]: Both me and @olizilla did not get initially that `preview_id` and `id` should be the same here

**Pull request merged**